package.json

```diff
@@ -136,12 +138,13 @@
     "testcafe-browser-tools": "2.0.23",
     "testcafe-hammerhead": "24.5.18",
     "testcafe-legacy-api": "5.1.4",
-    "testcafe-reporter-dashboard": "0.2.5",
+    "testcafe-reporter-dashboard": "0.2.6-rc.3",
     "testcafe-reporter-json": "^2.1.0",
     "testcafe-reporter-list": "^2.1.0",
     "testcafe-reporter-minimal": "^2.1.0",
     "testcafe-reporter-spec": "^2.1.1",
     "testcafe-reporter-xunit": "^2.2.1",
+    "testcafe-safe-storage": "^1.1.1",
```

Comment:
(minor)
The registration process requires pressing the Enter key after email and token entering.
But it's not required after pressing 'y' for changing default token process
![image](https://user-images.githubusercontent.com/12034551/166481938-ff40c7b1-013c-46ee-92f7-8bbb62e88c63.png)
Should we make it more consistent or it's a common way to press y/n without enter?

Reply:
The is a built-in logic of [prompts controls](https://github.com/terkelg/prompts#-types): your need to accept a value of the `text` control bypassing the `Enter` key and you don't need an additional key for accepting the `confirm` control value.

Theoretically, it's possible to do it using the [onState](https://www.npmjs.com/package/prompts#confirmmessage-initial) API. After discussing it in private with @helen-dikareva, we decided to not do it.